### PR TITLE
Apply contribution guidelines to "VideoUrl" rule

### PR DESCRIPTION
**Pull request by danilosilva87**

### CODE REVIEW

File: tests/unit/Rules/VideoUrlTest.php [hunk 1]
```diff
@@ -13,104 +13,56 @@
...
-            [null, 'example.com'],
-            [null, 'ftp://youtu.be/netHLn9TScY'],
-            [null, 'https:/example.com/'],
-            [null, 'https:/youtube.com/'],
-            [null, 'https://vimeo'],
-            [null, 'https://vimeo.com71787467'],
-            [null, 'https://www.google.com'],
-            [null, 'tel:+1-816-555-1212'],
-            [null, 'text'],
+            'vimeo service with subdomain' => [new VideoUrl('vimeo'), 'https://player.vimeo.com/video/71787467'],
```
Comment (batusa) [inline tests/unit/Rules/VideoUrlTest.php]: Nice descriptions 👍

File: library/Rules/VideoUrl.php [hunk 2]
```diff
@@ -14,8 +14,18 @@
...
 
-class VideoUrl extends AbstractRule
+/**
+ * Validates if the input is a video URL value:
+ *
+ * @author Danilo Correa <danilosilva87@gmail.com>
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author Ricardo Gobbo <ricardo@clicknow.com.br>
+ */
+final class VideoUrl extends AbstractRule
```
Comment (batusa) [inline library/Rules/VideoUrl.php]: I don't think the `$service` attribute is used anywhere. Could you drop it?

**Comment by batusa** [comment]
I just saw that this one is also at https://github.com/Respect/Validation/pull/1151

File: library/Rules/VideoUrl.php (hunk 2 above)
Comment (danilosilva87) [reply library/Rules/VideoUrl.php]: I also think, but when I delete this attribute the integration tests have failed, it is referenced in the exceptions.

**Comment by batusa** [comment]
Fixed at https://github.com/Respect/Validation/pull/1159

Thanks for contributing! 🐼

**Pull request closed without merging**